generators/bdk/python/templates/bot-app/gif_activities.py.ejs

```diff
@@ -1,38 +1,35 @@
-from jinja2 import Environment, FileSystemLoader, select_autoescape
+import os
+
+from jinja2 import Template
 
 from symphony.bdk.core.activity.command import SlashCommandActivity, CommandContext
 from symphony.bdk.core.activity.form import FormReplyActivity, FormReplyContext
 from symphony.bdk.core.service.message.message_service import MessageService
 
 
 class GifSlashCommand(SlashCommandActivity):
-    """Sends a form when sending @bot-name /gif
-    """
+    # Sends a form when sending @bot-name /gif
 
     def __init__(self, messages: MessageService):
-        super().__init__("/gif", True, self.display_gif_form)
+        super().__init__("/gif", False, self.display_gif_form)
         self._messages = messages
-
-        env = Environment(loader=FileSystemLoader(['<%= resourcesFolder %>']),
-                          autoescape=select_autoescape(['html', 'xml']))
-        self.template = env.get_template('gif.xml')
+        self.template = Template(open('resources/gif.jinja2').read())
```

Comment:
Should we have `autoescape=True` in the Template constructor ?

Reply:
Added